src/paths/environments/fetchAll.yaml

```diff
@@ -4,6 +4,9 @@ tags:
 operationId: environmentsFetchAll
 security:
   - bearerAuth: ["ENVIRONMENTS_READ"]
+parameters:
+  - $ref: "../../parameters/query/pagination/Page.yaml"
+  - $ref: "../../parameters/query/pagination/Size.yaml"
 responses:
   '200':
     description: Expected response to a valid request
```

Comment:
"../../schemas/environments/EmbeddedEnvironmentDto.yaml"
should be modified as well. Did you visually confirm the layout?

Reply:
response does not contain pagination information. have a look at src/paths/scripts/fetchAll.yaml for inspiration

Reply:
src/schemas/scripts/PagedScriptDto.yaml